### DISCUSSION TITLE
如何链接多个块

### DISCUSSION THREAD

**Discussion by Tang-GNG** [obsidianzh/forum#51] [Q&A常见问题]
想要链接一篇文章里的多个块，我试了一下好像只能一个块一个块的来，没有办法链接多个块。

**Comment by ghuserac** [comment]
同问这个问题，感觉obsidian的块很死板，基本是就是一行一个块。
可现实中描述一个问题真的只需一行吗？
很常见的，比如一个代码块，然后再通过几行有序/无序列表来描述解释这个代码块。
在obsidian中怎么都不可能让它们是同一个块（可事实上这几个块都是描述 同一个问题）。
要是一行行的引用真是麻烦到蛋疼。
能自定义块吗？块的起始上个标记，块的结尾上个标记，像代码块类似的逻辑。
